ci/k8s/deployment.yml

```diff
@@ -121,6 +121,65 @@ spec:
               key: django-secret-key
         - name: ENABLE_STATS
           value: "yes please"
+      - name: rsr-reports
```

Comment:
YAML is crazy. Maybe you want to remove some of the duplication using references https://blog.daemonl.com/2016/02/yaml.html

Reply:
woah! That is indeed crazy. I removed some of the duplication using references.

Reply:
Where you able to test that the new YAML is correct?

Reply:
I used this to verify the yaml. 
`python -c 'import sys, yaml, json; json.dump(yaml.load(sys.stdin), sys.stdout, indent=4)' < ci/k8s/deployment.yml | less`

The training yaml was tricky because it is a template. I just ended up deleting all the template tags and verifying

Reply:
Very clever!

The training yaml can be tested by deploying a new training env. Do you mind doing that?

Reply:
Sure, I can do that!

Reply:
I've tested this, and deployed it on `rsr4`. It seems to work!